### ISSUE TITLE
Arabic numerals as 'page numbers' in a web-book. 

### ISSUE THREAD

**Issue by paradisojr** [pressbooks/ideas#5]
- [x ] User interface (reading)

Faculty members at my university would like to see a page-numbering system (e.g. [Arabic numerals](https://en.wikipedia.org/wiki/Arabic_numerals)) made available in the web-book version so their students can properly cite where they find content for their essays. 

I imagine multiple disciplines, especially those that involve 'citing the required textbook' for class assignments, essays, et al., could benefit from such an add on. 

Something like this (below) from Scribd might be interesting:
<img width="1237" alt="screen shot 2018-03-06 at 4 14 33 pm" src="https://user-images.githubusercontent.com/33466325/37058802-a2944ee8-2159-11e8-8ce7-b98072822bd6.png">

Thanks!

Jim

**Comment by colomet** [comment]
You mean, to have the the current Display part and chapter numbers flexible? like to have it in the current place or in the footer of the page?

![captura de pantalla 2018-03-08 a las 20 48 26](https://user-images.githubusercontent.com/3929177/37172808-2afa6e02-2312-11e8-8bad-2c7b9c74e46b.png)

**Comment by paradisojr** [comment]
Sure. I don't have that "footer" checkbox in the Open Textbooks theme, but if there could be an additional item that showed: "**display page number in the footer** (or header for that matter)", that would suffice.

**Comment by colomet** [comment]
that checkbox is a photoshop example.

Maybe is a good idea to have that in the footer. As a way to uderstand how many pages have the book and the current position.  But not just the page number, also the total number of pages.

About your problem. If you have to have the citation. As is a digital book, the page place can change in the future. Maybe the best way is to copy and paste the url as it will never change.

**Comment by paradisojr** [comment]
You make a good point. I think my initial screenshot (example) would be a workable model, where **total pages** and **percent read** were available in the footer.

From a practical standpoint, the URL idea makes sense; however, faculty want to encourage proper citations amongst the student body (according to MLA or APA formatting guidelines), so the URL idea would not be a scalable / long-term solution.

**Comment by colomet** [comment]
> however, faculty want to encourage proper citations amongst the student body (according to MLA or APA formatting guidelines), so the URL idea would not be a scalable / long-term solution.


Digital books and physical books have differences. We can not use the same aproach. I will ask MLA

**Comment by colomet** [comment]
https://library.douglascollege.ca/-/media/777DE0E330304F6594857F8A2D358035.ashx?la=en

![captura de pantalla 2018-03-09 a las 9 17 29](https://user-images.githubusercontent.com/3929177/37197341-bc6955e8-237a-11e8-85d2-fa6890fcdbec.png)

**Comment by paradisojr** [comment]
Those numbers referenced in the **In-Text** row (e.g. Barrett **189**) are page numbers. 

That being said, MLA and APA offer caveats for eBooks with no page numbers or inconsistent page numbering per device/unfixed (as can be seen [here](http://purchase.libanswers.com/faq/36720)); however, **1)** those rules are a bit more complex and **2)** meeting the request for page numbers for my faculty go beyond citations. (That was just one example.) 

They would also like to be able to reference the eBook in their comments to students, so having page numbers facilitates that. _(EX: Take a look at page 2 for further explanation on _________ topic.)_

In the end, maybe including page numbers in eBooks is more trouble than it's worth; I am not an expert on that topic. I just want to look for a solution that would keep my faculty using the eBook (particularly to encourage the dynamism offered via plugin integration et al.).

**Comment by colomet** [comment]
pages in an ebook are not possible as each reader offer the content with a different number of pages.  Digital pages are dinamical, so it change with the screen, the typo, the size of typo ...

other solution is to use the chapter as number. As it will change less (but it can change anyway if the book have an update, but in that situation, you know is not the same version of the book)

**Comment by paradisojr** [comment]
Yeah. I get it. I suppose these faculty can just use the PDF version, but then we lose some of the dynamism provided through the platform.

**Comment by SteelWagstaff** [comment]
@paradisojr Faculty may be interested in using annotation (via Hypothesis) to create links/anchors to different parts of a book. It lets you point to parts of a text with even more precision/specificity than page numbers in a traditional citation. Here's an example: https://hyp.is/P4y0wiy0EemdG5eICrF_7g/web.hypothes.is/quick-start-guide/

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 8y ago with no updates in the last 6y 3m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
